addons/common/stringtable.xml

```diff
@@ -467,6 +512,7 @@
             <Polish>Wybierz preferowany arsenał używany przez Zeus Enhanced.\nJeśli arsenał ACE nie będzie dostępny to uruchomi się podstawowy wirtualny arsenał BI.</Polish>
             <Russian>Выбор предпочитаемого Арсенала\nlfЕсли не установлен ACE, будет запущен стандартный Виртуальный Арсенал.</Russian>
             <Japanese>Zeus Ehnhanced により優先された武器庫を選択します。\nACE3 武器庫が有効化されていない場合、ゲーム標準のバーチャル武器庫へ戻されます。</Japanese>
+            <Korean>ZEN에서 쓰일 아스널을 선택합니다. \n만약 ACE 아스널이 설치되어있지 않다면, BI 아스널로 변경될 것입니다.</Korean>
```

Comment:
Please use `Zeus Enhanced` here rather than ZEN.

Reply:
OK. I fixed it.